packages/wonder-blocks-testing/src/harness/adapters/router.tsx

```diff
@@ -48,7 +47,7 @@ type Config = // The initial location to use.
                     /**
                      * The location to use.
                      */
-                    location: string | Location;
+                    location: LocationDescriptor;
```

Comment:
**note:** This type encompasses the idea of a string or the shape of a location, which I believe is what we need here.